Robus/src/msg_alloc.c

```diff
@@ -397,7 +417,7 @@ static inline void MsgAlloc_ClearMsgTask(void)
 
     for (uint16_t rm = 0; rm < msg_tasks_stack_id; rm++)
     {
-        //LuosHAL_SetIrqState(true);
+        LuosHAL_SetIrqState(true);
```

Comment:
do we previously comment this line by mistake?
If yes we should make it in a separated commit to fixup the mistake on the original commit.